### PR TITLE
Data Types Assignment

### PR DESCRIPTION
**Pull request by lesleysab**
## Checkpoint Rubric
This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

**Checkpoint 1**
- [ ] All tests passed: 40 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Properly indented code: 15 points
- [ ] Demonstrated effective use of JavaScript: 30 points

**Checkpoint 2**
- [ ] The application works as it should: 40 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Properly indented code: 15 points
- [ ] Demonstrated effective use of JavaScript and the DOM API: 30 points

**Checkpoint 3**
- [ ] Use of React: 25 points
- [ ] Accesses an API: 25 points
- [ ] Proper use of documentation (commenting on code): 25 points
- [ ] The application functions as it should: 25 points

### CODE REVIEW

**Comment by lesleysab** [comment]
Lesley Sablatura Pull Request

File: 01week/datatypes.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  return num1 + num2
+}
+sumTwoNumbers(4, 6);
+
+// Write a JavaScript function that runs only when 2 things are true.
+//
+// &&
+
+function twoTruths(num1, num2) {
+  if (num1 <= 5 && num2 >= 10)
```
Comment (stephsingler) [inline 01week/datatypes.js:64]: This function evaluates if your two arguments are equal to a number that is true. I need the function to evaluate if both arguments are truthy, not equal to something true.

File: 01week/datatypes.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    return ('false');
+}
+twoTruths(6, 11);
+
+// Write a JavaScript program that runs when 1 of 2 things are true.
+//
+// ||
+
+function orTrue(num1, num2) {
+  if (num1 <= 5 || num2 >= 10)
```
Comment (stephsingler) [inline 01week/datatypes.js:76]: This function evaluates if one of your arguments is equal to something true. I need the function to evaluate if one argument is truthy.

File: 01week/datatypes.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+// Write a JavaScript program that runs when both things are not true
+//
+// !&&!
+
+
+function noTrue(num1, num2) {
+  if (!num1 == 5 || !num2 == 10)
+    return ("true");
+  else
+    return ('false');
```
Comment (stephsingler) [inline 01week/datatypes.js:93]: This function evaluates if your two arguments are equal to a false number. I need the function to evaluate if both arguments are falsey, not equal to a false value.

**Review by stephsingler — CHANGES_REQUESTED** [review]
Great start just look at the last 3 comments and make sure your function are evaluating...not checking true/false values

**Pull request closed without merging**